### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by t-bast**
Remove mentions of the demo GUI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,7 +32,7 @@ For more information please visit the [API documentation website](https://acinq.
 
 ## Documentation
 
-Please visit our [wiki](https://github.com/acinq/eclair/wiki) to find detailed instructions on how to configure your
+Please visit our [docs](https://github.com/lightningnetwork/lightning-rfc/issues/769) and [wiki](https://github.com/acinq/eclair/wiki) to find detailed instructions on how to configure your
```
Comment (pm47) [inline README.md:35]: Copy paste error?

Comment (t-bast) [reply README.md:35]: Wow oh yeah that went very wrong

Comment (pm47) [reply README.md]: Is there a reason why you are not using a relative link?

Comment (t-bast) [reply README.md]: I think you can't have a relative link to a folder, only to a file, can you?

Comment (t-bast) [reply README.md]: But I can add an index/readme file inside the `docs` folder and point to that if you prefer

Comment (pm47) [reply README.md]: Apparently you can (see https://github.com/ACINQ/eclair/tree/readme-relative-folder)

Comment (t-bast) [reply README.md]: Nice!

**Pull request merged**